### PR TITLE
Completed Typescript Template Literal Exercise

### PR DESCRIPTION
**Pull request by allygreven**
Screenshot 2024-10-22 at 9 48 31 AM (2)

### CODE REVIEW

File: typescript-template-literals/main.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const age: number = 25;
+
+const bio: string = `My name is ${firstName} ${lastName} and I am ${age} years old`;
+
+console.log('Bio:', bio);
+
+const fruit: string = 'apple';
+const item: string = 'pen';
+
+const applePen: string = `${fruit} ${item}`;
```
Comment (shawnkost) [inline typescript-template-literals/main.ts:18]: We actually want this without a space in-between the words!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looks good, just one very small edit here!

**Comment by allygreven** [comment]
Screenshot 2024-10-22 at 9 58 38 AM (2)

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**